input/pagecontent/StructureDefinition-ViewDefinition-notes.md

```diff
@@ -536,9 +445,30 @@ Another use case may be for users to select database-specific numeric types.
 Behavior is undefined and left to the runner if the expression returns a value
 that is incompatible with the underlying database type.
 
-### Processing Algorithm (Model)
+## Using Constants
+A ViewDefinition may include one or more of constants, which are simply values that can be reused
```

Comment:
Indentation on this line needs to be reduced to match the previous line.

Reply:
Can you clarify? Which line isn't indented properly?

Reply:
Sorry, I think I must have written my comment against the wrong line.

I have made a commit to fix this: afb96053c44bf1b44d34e32979a26767f1e861e7